src/dnf-plugins/product-id/test-product-id.c

```diff
@@ -466,7 +474,12 @@ void teardownInstalledProduct(installedProductCertsFixture *fixture, gconstpoint
 void testInstalledProduct(installedProductCertsFixture *fixture, gconstpointer testData) {
     (void)testData;
 
-    int ret = getInstalledProductCerts("./test_data/cert_dir/", fixture->repos, fixture->enabledRepoProductId, fixture->productDb);
+    int ret = getInstalledProductCerts(
+            "./test_data/cert_dir/",
+            fixture->repos,
+            fixture->enabledRepoProductId,
+            fixture->productDb
+            );
```

Comment:
not needed

Reply:
Do you mean that this unit test is not needed or do you like very long lines?

Reply:
I personally prefer the format change :)

Reply:
> I personally prefer the format change :)

Sure... in _its own_ commit/PR, not stuffed together "static analysis fixes"